frontend/lib/identification/verification_workflow/verification_qr_content_parser.dart

```diff
@@ -12,16 +12,14 @@ class VerificationParseException extends QrCodeParseException {
       : super(internalMessage);
 }
 
-extension QRParsing on String {
+extension QRParsing on Uint8List {
```

Comment:
It feels wrong that this is an extension on Uint8List. Also, it seems this is not just a verification_qr_content_parser, but also a general qr_code_parser. Maybe you can move this functionality to qr_code_utils.

Reply:
We can do that in another PR. and generally refactor things.